pkg/server/server.go

```diff
@@ -434,17 +439,24 @@ func (s *Server) HTTPHandler() (http.Handler, error) {
 	)
 
 	// List operator operands endpoint
-	operandsListHandler := &OperandsListHandler{
+	operatorHandler := &OperandsListHandler{
 		APIServerURL: k8sProxyURL,
 		Client: &http.Client{
 			Transport: internalProxiedK8SRT,
 		},
 	}
 
+	handle(operatorListEndpoint, http.StripPrefix(
+		proxy.SingleJoiningSlash(s.BaseURL.Path, operatorListEndpoint),
+		authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
+			operatorHandler.OperatorListHandler(user, w, r)
+		}),
+	))
+
 	handle(operandsListEndpoint, http.StripPrefix(
 		proxy.SingleJoiningSlash(s.BaseURL.Path, operandsListEndpoint),
 		authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
-			operandsListHandler.OperandsListHandler(user, w, r)
+			operatorHandler.OperandsListHandler(user, w, r)
 		}),
 	))
```

Comment:
IMO, as we continue to add more specialized handlers to our server, we should be splitting them off into separate concerns with their own multiplexed handlers. So for operator related endpoints, we would just have an "OperatorHandler" type, with a "ServeHTTP" method, and that method would return an http.ServeMux (just like our main server). This muliplexed handler would handle all requests prefixed with `/api/bridge/operators` or something, and would define handlers for all endpoints of that sub-tree. Then, here in the main server, all we have to do is pass all requests with that prefix to the OperatorHandler instance.